test/vectors/longtime_comformance_test.cfg

```diff
@@ -0,0 +1,12 @@
+# Use following format to define a test vector
+#
+# file_name Y4M? color_format w h bit_depth compressed_10bit_output start_frame frame_count
+#
+# Lines start with # will be ignored.
+# File must saved in same folder with config file.
+# color_format support 420, 420p10 in current version.
+# If 'compressed_10bit_output' set on with a 420p10 color format input file, test app will feed video frame into encoder with compressed 10 bit format.
+# start_frame and frame_count will set test range of a input file.
+# With 'frame_count = 0', will use all following frames start from 'start_frame'.
+# start_frame use 0 as first frame index.
+kirland_640_480_30.yuv 0 420 640 480 8 0 0 360
```

Comment:
just a double check for the length of 360 frames seems a little bit short for a "longtime" test.

Reply:
Yes, I just want to test it.:)